incubator/group/keeper.go

```diff
@@ -35,8 +38,19 @@ const (
 	VoteByVoterIndexPrefix        byte = 0x42
 )
 
+type ProposalI interface {
```

Comment:
Pretty sure what I suggested before will work better:
```suggestion
type ProposalI interface {
  orm.Persistent
  GetBase() ProposalBase
  SetBase(ProposalBase)
  GetMsgs() []Msg
  SetMsgs([]Msgs) error
}
```

Reply:
I think I still have not fully understood your design, sorry. Can you elaborate on the workflow you have in mind with `Get/SetMsgs` in the proposal type? Are `Msg` the `sdk. Msg` or some other types?

I assume with `orm.Persistent` you want the obj stored but with the typed `proposalTable` we would need a single container like current `MyAppProposal` with `oneof` that implements `ProposalModel`.